router/tcp.go

```diff
@@ -34,6 +34,21 @@ type TCPListener struct {
 	closed   bool
 }
 
+func (s *TCPListener) List() ([]*router.Route, error) {
```

Comment:
I can't figure out why this function was added. Doesn't look like it's being used anywhere?